app/models/concerns/spotlight/translatables.rb

```diff
@@ -16,7 +16,13 @@ def translates(*attr_names)
       # and defaults to the ActiveRecord value
       def define_translated_attr_reader(attr_name)
         define_method(:"#{attr_name}") do
-          I18n.translate(attr_name, scope: slug, default: attr_translation(attr_name))
+          send("translated_#{attr_name}", default: [attr_translation(attr_name), ''])
+        end
+
+        define_method(:"translated_#{attr_name}") do |default: [], **options|
```

Comment:
```suggestion
        # Define an accessor that gets the value of the attribute in a given locale,
        # returning `nil` for untranslated values. 
        #
        # Note: For the default locale, we actually want to dig into the database,
        # because that is the source of truth for the data.
        define_method(:"translated_#{attr_name}") do |default: [], **options|
```